server.go

```diff
@@ -487,3 +467,12 @@ func (s *Server) logf(format string, v ...interface{}) {
 		s.cfg.Logger.Printf(format, v...)
 	}
 }
+
+func (s *Server) isServing() error {
+	s.mu.RLock()
+	defer s.mu.RUnlock()
+	if !s.serving {
```

Comment:
I don't see where this is ever set to false?

Reply:
@epsniff The zero value for booleans is `false`, so we don't need to explicitly set it.

Reply:
Oh, I assumed this func was used during the Stop part of the life cycle and was used to indicate the system was shutting down.

Reply:
No, it’s used in NewMailbox() to ensure unstarted servers don’t create mailboxes.